<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Qwt Plot</string>
    </attribute>
    <widget class="QwtPlot" name="Grafica">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>941</width>
       <height>411</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="canvasBackground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="autoReplot">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QwtKnob" name="frecuencia">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>711</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Este ejemplo muestra cómo pintar curvas en una gráfica de Qt. Las curvas que se pintan son unas sinusoides calculadas según una fórmula. El periodo de las sinusoides se puede cambiar girando la rosca (para demostrar cómo se repintan las gráficas). Esto es símplemente un ejemplo, las curvas que hay que pintar al realizar la práctica son las muestras obtenidas con el conversor analógico digital por la Tiva y mandadas hacia Qt. Se pintará una curva por cada canal de entrada (4 en total). La gráfica se actualizará conforme vayan llegando nuevas muestras, descartando (borrando) las muestras más antiguas. Cuando lleguen muestras nuevas no sólo se pintarán las que hayan llegado, sino también las anteriores. Cada curva pintada contendrá al menos las últimas 1000 muestras. Al arrancar la funcionalidad, las muestras estarán inicializadas a 0 y se irán actualizando con los valores que vayan llegando. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QHexEdit" name="hexEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>831</width>
       <height>281</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>QHexEditWidget</string>
     </property>
     <property name="whatsThis">
      <string>QHexEdit widget allow to edit the data in hex view.</string>
     </property>
     <property name="asciiArea">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <longLong>0</longLong>
     </property>
     <property name="data">
      <cstring></cstring>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QNeedleIndicator" name="qneedleindicator">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>340</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="Knob_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Widgets Varios</string>
    </attribute>
    <widget class="Led" name="led">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>90</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
    </widget>
    <widget class="PotentioMeter" name="dial">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
    <widget class="ThermoMeter" name="thermometer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>40</width>
       <height>160</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Shows the pressure</string>
     </property>
     <property name="whatsThis">
      <string>The bar meter widget displays the pressure attached to it</string>
     </property>
    </widget>
    <widget class="color_widgets::ColorWheel" name="colorWheel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QwtKnob" name="Knob">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QDigitalGauge" name="qdigitalgauge">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enciende</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="color_widgets::ColorPreview" name="colorPreview">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="Counter" name="counter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
    <widget class="QLedMatrix" name="ledMatrix">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>300</y>
       <width>411</width>
       <height>100</height>
      </rect>
     </property>
     <property name="backgroundColor">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="backgroundMode">
      <enum>Qt::OpaqueMode</enum>
     </property>
     <property name="darkLedColor">
      <color>
       <red>34</red>
       <green>34</green>
       <blue>34</blue>
      </color>
     </property>
     <property name="rows" stdset="0">
      <number>16</number>
     </property>
     <property name="columns" stdset="0">
      <number>64</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>QCharts</string>
    </attribute>
    <widget class="QChartView" name="chartview">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QChartView" name="chartview_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>871</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QChartView" name="chartview_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QChartView" name="chartview_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="Knob_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>110</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>240</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>La porcion de Joe cambia al soltar la rosca</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ThermoMeter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/thermometer.h</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>PotentioMeter</class>
   <extends>QAbstractSlider</extends>
   <header>analogwidgets/potentiometer.h</header>
  </customwidget>
  <customwidget>
   <class>Counter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/counter.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>QDigitalGauge</class>
   <extends>QWidget</extends>
   <header>qdigitalgauge.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
  <customwidget>
   <class>QLedMatrix</class>
   <extends>QWidget</extends>
   <header>qledmatrix.h</header>
  </customwidget>
  <customwidget>
   <class>QNeedleIndicator</class>
   <extends>QWidget</extends>
   <header>qneedleindicator.h</header>
  </customwidget>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QAbstractScrollArea</extends>
   <header>qhexedit.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Knob</sender>
   <signal>valueChanged(double)</signal>
   <receiver>qdigitalgauge</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>thermometer</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>led</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorWheel</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>colorPreview</receiver>
   <slot>setColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>counter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Knob_3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>qneedleindicator</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
